doc/user-guide/data_model.rst

```diff
@@ -647,47 +651,48 @@ cannot be made so because field data is not expected to match the
 number of cells or points.  As such, it also cannot be plotted.
 
 
-Vectors, Textures, and Normals Attributes
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Both cell and point data can also store the following "special" attributes in addition to :attr:`active_scalars <pyvista.core.dataset.DataSet.active_scalars>`
+Vectors, Texture Coords, and Normals Attributes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Both cell and point data can also store the following "special" attributes in addition to :attr:`active_scalars <pyvista.DataSet.active_scalars>`:
 
-* :attr:`active_normals <pyvista.core.dataset.DataSet.active_normals>`
-* :attr:`active_t_coords <pyvista.core.dataset.DataSet.active_t_coords>`
-* :attr:`active_vectors <pyvista.core.dataset.DataSet.active_vectors>`
+* :attr:`active_normals <pyvista.DataSet.active_normals>`
```

Comment:
`DataSet.active_normals` is not actually a thing, but it should be. I don't have the time right now to implement that too.

Reply:
I'm adding this in.